### ISSUE TITLE
RTOFS post scripts with CFP but not run under different sub-dir or no separated output logfiles (bugzilla #1133)

### ISSUE THREAD

**Issue by DanIredell-NOAA** [NOAA-EMC/RTOFS_GLO#12]
http://www2.spa.ncep.noaa.gov/bugzilla/show_bug.cgi?id=1133

Please address the RTOFS post scripts with CFP but not run under different sub-dir or no separated output logfiles.

The following scripts run with MPMD and CFP but not run under different sub-dir or no separated output logfiles - 
  scripts/exrtofs_glo_post.sh

Please refer to https://www.nco.ncep.noaa.gov/idsb/implementation_standards/Implementation%20Standards%20v10.2.pdf?, page 12, section "Working directories"

**Comment by DanIredell-NOAA** [comment]
Branch bugfixes: Directed output to separate log files for CFP in exrtofs_glo_post.sh

**Comment by DanIredell-NOAA** [comment]
Code change for this specific issue was this (scripts/exrtofs_glo_post.sh):

```
<        echo "${USHrtofs}/${RUN}_glo3z_6hrly.sh $reg " >> cmdfile
---
>        echo "${USHrtofs}/${RUN}_glo3z_6hrly.sh $reg > $reg.$fhr.out 2>&1" >> cmdfile

```

**Comment by DanIredell-NOAA** [comment]
NCO:
This is fixed in the RTOFS v2.4.3 codes/package.
